tests/jlm/hls/util/TestView.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2025 Magnus Sjalander <work@sjalander.com>
+ * See COPYING for terms of redistribution.
+ */
+
+#include "test-registry.hpp"
+
+#include <jlm/hls/backend/rvsdg2rhls/add-forks.hpp>
+#include <jlm/hls/backend/rvsdg2rhls/ThetaConversion.hpp>
+#include <jlm/hls/ir/hls.hpp>
+#include <jlm/hls/util/view.hpp>
+#include <jlm/llvm/ir/operators.hpp>
+#include <jlm/rvsdg/view.hpp>
+
+static int
+TestDumpDot()
+{
+  std::cout << std::endl << "### Test dump_dot ###" << std::endl << std::endl;
+
+  using namespace jlm;
+  using namespace jlm::hls;
+  using namespace jlm::llvm;
+
+  // Arrange
+  auto b32 = rvsdg::bittype::Create(32);
+  auto ft = rvsdg::FunctionType::Create({}, { b32 });
+
+  RvsdgModule rm(util::filepath(""), "", "");
```

Comment:
I believe that there is no need to create an RvsdgModule, as it is nowhere used. You could also just create a graph: `rvsdg::Graph graph`

Reply:
Changed to graph